server/constants/collectives.js

```diff
@@ -4,9 +4,12 @@ export const types = {
   COLLECTIVE: 'COLLECTIVE',
   EVENT: 'EVENT',
   USER: 'USER',
-  ORGANIZATION: 'ORGANIZATION'
+  ORGANIZATION: 'ORGANIZATION',
+  BOT: 'BOT'
 };
 
+export const OPENCOLLECTIVE_BOT_NAME = 'Open Collective bot';
```

Comment:
Let's do `Open Collective Bot`. Since you are creating a collective for it, I wouldn't have a constant for the name. Just save it in the database as `collective.name`.

Reply:
Now that I think of it, maybe we should call this bot "IRS Bot" (Internal Revue Service), since this particular feature is really about the IRS.

Reply:
sure. IRS coming along.